### PR TITLE
Functional specifications into main branch

### PR DESCRIPTION
**Pull request by Evan-UHRING**

### CODE REVIEW

File: Documents/FunctionalSpecifications/functionalSpecifications.md [hunk 1]
```diff
@@ -112,14 +113,14 @@ Make a recreation of the game Frogger while adding our own touch to it.
...
 | Each button of the FPGA has to be one of the four direction movement |
 | The game can be reset to the beginning by pressing the four buttons of the FPGA at the same time |
-| The display has to be a VGA display |
-| The level number has to be displayed using the two seven segment displays of the FPGA |
+| The display has to be a VGA<sup><a href="#3">[3]</a></sup> display |
+| The level number has to be displayed using the two seven segment displays<sup><a href="#4">[4]</a></sup> of the FPGA |
 
 | Out of scope |
 | --- |
 | Each level don't increase in difficulty |
```
Comment (EnzoGuillouche) [inline Documents/FunctionalSpecifications/functionalSpecifications.md]: Each level should be singular, the use of 'does not' is then necessary.

File: Documents/FunctionalSpecifications/functionalSpecifications.md [hunk 2]
```diff
@@ -49,6 +49,7 @@
         - [The Game](#the-game)
             - [The environment](#the-environment)
             - [The player](#the-player)
+        - [The Console](#the-console)
     - [4. Personas definition](#personas-definition)
```
Comment (Fus1onAxel) [inline Documents/FunctionalSpecifications/functionalSpecifications.md]: not linked (as well as cars)

**Review by Fus1onAxel — CHANGES_REQUESTED** [review]

File: Documents/FunctionalSpecifications/functionalSpecifications.md (hunk 2 above)
Comment (Evan-UHRING) [reply Documents/FunctionalSpecifications/functionalSpecifications.md]: cars yes it doesn't work but this one you showed works

File: Documents/FunctionalSpecifications/functionalSpecifications.md [hunk 3]
```diff
@@ -191,106 +193,137 @@ It uses the monospace sans-serif font in uppercase, the same font as the game Pa
...
+When going on it, the player gain extra points (see [points table](#scoring)).
 
 #### Lady frogs
 
-As well as the insect, a lady frog can spawn 
+As well as the insect, a lady frog can spawn randomly.
+
+<img src="Images/froggerLadyFrog.png" width="20%"><br>
+
+When the player steps on it, he grabs the lady frog, to escort it to the finish line. If the player successfully escorted the lady frog, the player gain extra points (see [points table](#scoring)).
```
Comment (EnzoGuillouche) [inline Documents/FunctionalSpecifications/functionalSpecifications.md]: the player gain*s*

File: Documents/FunctionalSpecifications/functionalSpecifications.md (hunk 2 above)
Comment (Evan-UHRING) [reply Documents/FunctionalSpecifications/functionalSpecifications.md]: fixed, i let you check if it's good

**Review by EnzoGuillouche — CHANGES_REQUESTED** [review]
Seems overall good. Some typos are remaining to be corrected tho.

File: Documents/FunctionalSpecifications/functionalSpecifications.md (hunk 2 above)
Comment (EnzoGuillouche) [reply Documents/FunctionalSpecifications/functionalSpecifications.md]: cars is alright, though personas definition still does not work

Comment (Fus1onAxel) [reply Documents/FunctionalSpecifications/functionalSpecifications.md]: persona not working

**Review by Fus1onAxel — CHANGES_REQUESTED** [review]

File: Documents/FunctionalSpecifications/functionalSpecifications.md (hunk 1 above)
Comment (Evan-UHRING) [reply Documents/FunctionalSpecifications/functionalSpecifications.md]: fixed

File: Documents/FunctionalSpecifications/functionalSpecifications.md (hunk 3 above)
Comment (Evan-UHRING) [reply Documents/FunctionalSpecifications/functionalSpecifications.md]: fixed and i found the same mistake some lines before that i also fixed

**Review by Fus1onAxel — APPROVED** [review]

**Review by EnzoGuillouche — APPROVED** [review]
All the seen issues have been fixed. Merging approved.

**Pull request merged**